SeriesMaster-Backend/routes/users.js

```diff
@@ -50,6 +50,22 @@ function createUserRouter(database) {
     }
   });
 
+  router.post("/pushtoken", async (req, res) => {
+    try {
+      const { id, token } = req.body;
+
+      if (!token) {
+        return res.send("no Expo token provided");
+      }
+
+      await collection.updateOne({ userId: id }, { $set: { token: token } });
+
+      res.send("OK");
+    } catch (error) {
+      console.log(error);
```

Comment:
Make sure to close the connection to the client. Right now, the connection stays active if an error is thrown.
e.g. `res.status(500).send(error.message)`